addons/calendar/data/mail_template_data.xml

```diff
@@ -22,7 +22,12 @@
         Hello <t t-out="object.common_name or ''">Wood Corner</t>,<br/><br/>
 
         <t t-if="not target_responsible">
-            <t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t> invited you for the <strong t-out="object.event_id.name or ''">Follow-up for Project proposal</strong> meeting.
+            <t t-if="not object.event_id.user_id or (object.event_id.user_id.partner_id.id == object.env.ref('base.partner_root').id)">
```

Comment:
Maybe we could use a better heuristic (that does not depend on master data), using archived (that also allows to simplify the whole condition to a single one), like
```suggestion
            <t t-if="not object.event_id.user_id.active">
```